skyd/skyd.go

```diff
@@ -121,6 +139,9 @@ func (c *client) PinnedSkylinks() (map[string]interface{}, error) {
 		}
 		// Grab all subdirs and queue them for walking.
 		for _, d := range rd.Directories {
+			if d.SiaPath.String() == dir.String() {
```

Comment:
For context, the current directory is always the 0 index of the slice of returned directories. 

Just for future reference of why this came up.

Reply:
I'll clean it up in the next F/U.